CHANGES.md

```diff
@@ -6,6 +6,10 @@ Change Log
 ##### Additions :tada:
 * Added optional `index` parameter to `PrimitiveCollection.add`. [#8041](https://github.com/AnalyticalGraphicsInc/cesium/pull/8041)
 
+##### Fixes :wrench:
+* Disable FXAA by default. [#7875](https://github.com/AnalyticalGraphicsInc/cesium/issues/7875)
+* Now multiply the default maximum screen space error by a users device pixel ratio. This changes effects 3DTilesets, Imagery and items depending on Globe SSE. [#7875](https://github.com/AnalyticalGraphicsInc/cesium/issues/7875)
```

Comment:
This is both too high level and too in the weeds, we need to frame it in a way end users care about.  Consider splitting it into two bullets with something like:

> * Improve label legibility and overall visual quality by rendering at device native resolution regardless of the `window.devicePixelRatio` setting. [#7875](https://github.com/AnalyticalGraphicsInc/cesium/issues/7875)
>
>    To go back to the old behavior:
>
>     <code_ snippet>
>
> * Updated default values for `globe.maximumScreenSpaceError` and `Cesium3DTileset.maximumScreenSpaceError` to improve bandwidth and memory usage on devices with `window.devicePixelRatio` values greater than 1. [#7875](https://github.com/AnalyticalGraphicsInc/cesium/issues/7875)
>
>    To go back to the old behavior:
>
>     <code_ snippet>
>